entity-framework/core/querying/null-semantics.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Query null semantics in EF Core
+description: Information on how Entity Framework Core handles null comparisons in queries 
+author: maumar
+ms.date: 11/11/2020
+uid: core/querying/null-semantics
```

Comment:
null-comparisons also rename file.

We refer to null semantics but let's use a more understandable word?